.github/scripts/translation/make_translation.sh

```
@@ -0,0 +1,164 @@
+    "pt_PT"
+)
+
+#
+# Define global vars
+#
+PO_SRC=$BASE_DIR/po_src
+CAN_BE_TRANSLATE=false
+
+# Check project name
+if [ "$1" ]; then
+    for PROJECT in ${PROJECTS[@]};
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)